src/App.css

```diff
@@ -1,15 +1,14 @@
 .inputCategory {
-    background-color: rgb(246, 217, 217);
-    max-width: 280px;
-    min-width: 240px;
-    margin-top: 24px;
-    height: 38px;
-    border-radius: 6px;
-    padding-left: 10px;
-  }
-  
-  .categoryContainer {
-    display: flex;
-    justify-content: center;
-  }
-  
\ No newline at end of file
+  background-color: rgb(246, 217, 217);
+  max-width: 280px;
+  min-width: 240px;
+  margin-top: 24px;
+  height: 38px;
+  border-radius: 6px;
+  padding-left: 10px;
+}
+
+.categoryContainer {
```

Comment:
[OPTIONAL] This is not the correct way of naming your Css selectors. Kindly consider changing it to `hyphen delimited string` like `category-container` instead of the camel case naming convention which is predominant in JavaScript naming. :wink:

Reply:
Hello @jerryowusu,

Thank you for taking the time to review my work. I appreciate your efforts to help me improve.

However, I'm sorry to hear that you're experiencing difficulties with my code. On my end, everything seems to be working fine, so I'm not sure what could be causing the problem on your system.

Please let me know if there's anything else I can do to assist you, or if you have any suggestions on how we can resolve this issue.

Thank you again for your feedback.